docker-compose.yml

```diff
@@ -1,18 +1,24 @@
-version: "2.0"
+version: '3'
 services:
   titra:
     image: kromit/titra
-    container_name: titra
+    container_name: titra_app
     depends_on:
       - mongodb
     environment:
       - ROOT_URL=${ROOT_URL}
       - MONGO_URL=mongodb://mongodb/titra?directConnection=true
       - PORT=3000
     ports:
-      - "3000:3000"
+      - "127.0.0.1:3000:3000"
     restart: always
   mongodb:
     image: mongo:5.0
-    container_name: mongodb
+    container_name: titra_db
     restart: always
+    environment:
+      - MONGO_DB:titra
+    volumes:
+      - titra_mongodb_volume:/data/db
```

Comment:
use a named volume

Reply:
shouldn't this be named titra_db_volume to stay aligned with the naming of the mongodb container?

Reply:
yeah right